package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-farmer-claim",
-  "version": "0.35.1",
+  "version": "0.35.2",
```

Comment:
Probably should be a minor bump up to 0.36.0

Reply:
bumped